core/amber/src/main/scala/edu/uci/ics/texera/web/model/websocket/event/CacheStatusUpdateEvent.scala

```diff
@@ -1,6 +1,3 @@
 package edu.uci.ics.texera.web.model.websocket.event
 
-import edu.uci.ics.texera.web.model.common.CacheStatus
-
-case class CacheStatusUpdateEvent(cacheStatusMap: Map[String, CacheStatus])
-    extends TexeraWebSocketEvent
+case class CacheStatusUpdateEvent(cacheStatusMap: Map[String, String]) extends TexeraWebSocketEvent
```

Comment:
why downgrade `CacheStatus` to String?

Reply:
`CacheStatus` is an enum class generated by protobuf. If we use the existing JSON serializer on this class, the frontend won't recognize it.